src/runtime/flags.ts

```diff
@@ -30,7 +30,7 @@ class FlagDefaults {
   static warnOnUnsafeRefinement = false;
   // Enables support for nullable/optional types in JS and Kotlin code generation.
   // See b/174115805 for more info.
-  static supportNullables = false;
+  static supportNullables = true;
```

Comment:
Can you call this out in the CL description please?

Reply:
Done